analysis/graph/db_helper.py

```diff
@@ -0,0 +1,21 @@
+"""Utility function to load/reload graph for development."""
+from metakb.database import Graph
+from metakb import PROJECT_ROOT
+import json
+
+
+g = Graph(uri="bolt://localhost:7687", credentials=("neo4j", "admin"))
+g.clear()
+
+fpath = PROJECT_ROOT / 'analysis' / 'graph' / 'civic_cdm_latest.json'
```

Comment:
If we keep this, wondering if we could just have it point to the file in `/data/tranform/civic_cdm.json`

Reply:
Maybe we could just fold this functionality into the `cli` module and grab from the respective `/data/transform` folders -- I'll add to issue-59